### DISCUSSION TITLE
How to install python in vs code

### DISCUSSION THREAD

**Discussion by kabir33777** [coder/code-server#3972] [Q&A] [answered]
can anyone help how to install python

**Comment by jsjoeio** [comment]
What OS are you using?

**Reply by kabir33777** [reply]
i am using android vscode

**Reply by kabir33777** [reply]
i know you can download the python interpreter but how

**Reply by lg-nineoneninethreenine** [reply]
Wait what? This guide was for code-server. Not for Android VSCode. First of all, there is nothing called Android VSCode. If you are using Termux, get code-server on Linux and then, try activating SSH.

**Reply by kabir33777** [reply]
I am using vheditor which uses code-server go on playstone and check it(for android 10 + use there github page for dwnload)

**Reply by lg-nineoneninethreenine** [reply]
Use WSL.

**Reply by lg-nineoneninethreenine** [reply]
Or Linux.

**Reply by lg-nineoneninethreenine** [reply]
> I am using vheditor which uses code-server go on playstone and check it(for android 10 + use there github page for dwnload)

Do not download ANYTHING from the Play Store. Instead, use the Windows Subsystem for Linux.

**Reply by kabir33777** [reply]
ok but how

**Reply by lg-nineoneninethreenine** [reply]
Follow this: https://github.com/cdr/code-server/discussions/3972#discussioncomment-1166350

@jsjoeio marked it as answer too

**Comment by lg-nineoneninethreenine** [comment]
I think you try installing Anaconda 3 which gives Python 3.8 built in.

**Reply by lg-nineoneninethreenine** [reply]
And use code-server in WSL if Windows or Terminal on Macs and Linux computers.

**Reply by lg-nineoneninethreenine** [reply]
Did that yesterday. If you successfully installed it, on Linux and WSL, before the username and hostname, you will have `(base)` as a result of the base Conda environment. Try launching Python 3.8 via `python`. Worked

**Reply by lg-nineoneninethreenine** [reply]
If you want the steps, read this(works on WSL too):
1. Get the Anaconda Installer link from https://www.anaconda.com/products/individual. Right click the Linux installer and hit Copy Link.

2. Go to your Terminal, type `cd ~` and then, type this:
```bash
mkdir tmp
cd tmp
```
3. Use `wget`. As of now, the latest way to get Anaconda is:
```bash
wget https://repo.anaconda.com/archive/Anaconda3-2021.05-Linux-x86_64.sh
```
Then, type:
```bash
bash Anaconda3-2021.05-Linux-x86_64.sh
```
Accept everything and then, type:

```bash
cd ~
source .bashrc
```
Then, restart your terminal or your computer and open the Terminal. Now you must see it like so:
```
(base) user@hostname~$
```
Voila! Python is installed. Check it out and verify your installation with the classic Hello World line :)

Mainly, verify via running:
```
python
```
If those details mention Anaconda3 or Anaconda, you installed it successfully :)

**Reply by kabir33777** [reply]
no i am doing it on android with vheditor i want to install python interpretor on my android vscode

**Reply by lg-nineoneninethreenine** [reply]
You... can't.

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Use this method:

> If you want the steps, read this(works on WSL too):
> 1. Get the Anaconda Installer link from https://www.anaconda.com/products/individual. Right click the Linux installer and hit Copy Link.
> 
> 2. Go to your Terminal, type `cd ~` and then, type this:
> ```bash
> mkdir tmp
> cd tmp
> ```
> 3. Use `wget`. As of now, the latest way to get Anaconda is:
> ```bash
> wget https://repo.anaconda.com/archive/Anaconda3-2021.05-Linux-x86_64.sh
> ```
> Then, type:
> ```bash
> bash Anaconda3-2021.05-Linux-x86_64.sh
> ```
> Accept everything and then, type:
> 
> ```bash
> cd ~
> source .bashrc
> ```
> Then, restart your terminal or your computer and open the Terminal. Now you must see it like so:
> ```
> (base) user@hostname~$
> ```
> Voila! Python is installed. Check it out and verify your installation with the classic Hello World line :)
